src/utils/data.ts

```diff
@@ -462,7 +463,10 @@ export const getAttributeValue = (mxObject: MxO, attribute: string): Datum => {
         }
     }
     if (valueObject.isDate(attributeName)) {
-        return new Date(valueObject.get(attributeName) as number).toISOString();
+        const dateValue: number = valueObject.get(attributeName) as number;
+
+        // Converts the timestamp to ISO-8601
+        return asString(new Date(dateValue));
```

Comment:
Fine with the change, but is there no way to do this without a library? Altho the bundle size of the library is not big, it doesn't feel like it's doing a lot. But please correct me if I'm wrong and this isn't trivial to do on our own

Reply:
The libs is only 16kb. And actually only doing manual parsing can solve the problem as there is no default standard method to do this formatting.

Reply:
Had a quick chat with Diego, the library is already included in this widget anyways, so the library size is not really a consideration anymore.